tests/test_auth.py

```diff
@@ -1,9 +1,58 @@
+import io
+import json
 import unittest
-
+from unittest import mock
+from unittest.mock import patch
+import sys
 import kagglehub
+from kagglehub.config import get_kaggle_credentials
+from tests.fixtures import BaseTest
+
 
+class TestAuth(BaseTest):
 
-class TestAuth(unittest.TestCase):
-    def test_login(self):
-        with self.assertRaises(NotImplementedError):
+    def test_login_prompts_for_username(self):
```

Comment:
A user should always specify both the username and  the api key. They should not be able to specify only the username.